Last-Update: 2012-08-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Description: backported patch (r31749)
 Fix bug with copy & paste.
 The condition checks the wrong data (target instead of data).

--- a/src/af/xap/gtk/xap_UnixClipboard.cpp
+++ b/src/af/xap/gtk/xap_UnixClipboard.cpp
@@ -324,7 +324,7 @@
 		UT_DEBUGMSG(("Looking for %s on clipbaord \n",formatList[i]));
 		if(selection)
 		{
-			if (gtk_selection_data_get_target(selection) && (gtk_selection_data_get_length(selection) > 0))
+			if (gtk_selection_data_get_data(selection) && (gtk_selection_data_get_length(selection) > 0))
 			{
 			  if(!rval)
 			    {
